lms/static/sass/shared/_alerts.scss

```diff
@@ -0,0 +1,169 @@
+// lms alerts
+// This file contains "I want alerts that look like the pattern library" sass
+// and should be replaced when moving to the patter library proper
+// https://github.com/edx/ux-pattern-library/blob/master/pattern-library/sass/patterns/_alerts.scss
+
+// ------------------------------
+// edX Pattern Library: Utilities - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// #CONFIG
+// #UTILITIES
+// #GENERAL
+// #INDIVIDUAL CASES
+
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$alert-background-color: #fcfcfc;
+$alert-icon-color: #fcfcfc;
+$alert-border-radius: 0.3125rem;
+// border around the sides and bottom
+$alert-border: 0.0625rem solid #cdd7db;
+
+// alert colors
+$alert-information-color: #6fa0ba;
+$alert-warning-color: #fdbc56;
+$alert-error-color: #b20610;
+$alert-success-color: #25b85a;
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+@mixin alert($alert-color) {
+    border-top:  rem(2) solid $alert-color;
+
+    .alert-icon {
+        color: $alert-icon-color;
+        background-color: $alert-color;
+    }
+}
+
+@mixin alert-message($width) {
+    @include float(left);
+    width: $width;
+    padding: 1.25rem;
+    padding-top: 0;
+    padding-bottom: 0;
+
+    :last-child {
+        // keeps the message compact
+        margin-bottom: 0;
+    }
+}
+
+// everything below here gets added specificity pattern-library-shim
+.pattern-library-shim {
```

Comment:
@efischer19 I'm fine with this approach, but I would feel more comfortable if it was made explicit in the file name that this is a pattern library shim (so without opening the file you can tell that this is intended as a temporary file until we move across to the Pattern Library).